zerver/data_import/slack.py

```diff
@@ -463,21 +451,32 @@ def process_mpims(mpims: List[Dict[str, Any]]) -> None:
         mpims = []
     process_mpims(mpims)
 
-    for user in zerver_userprofile:
-        # this maps the recipients and subscriptions
-        # related to private messages
-        recipient = build_recipient(user['id'], recipient_id_count, Recipient.PERSONAL)
-        sub = build_subscription(recipient_id_count, user['id'], subscription_id_count)
+    for username in added_users:
```

Comment:
Namespace-wise, this should be `for slack_user_id in added_users`, since that's what type of "user" like thing these are.

Reply:
I added a new commit addressing this.